coq/coq-syntax.el

```diff
@@ -488,8 +488,12 @@ so for the following reasons:
     ("Instance" nil "Instance #:#.\nProof.\n#Defined." t "Instance")
     ("Program Instance" "pinstance" "Program Instance [ # ] => # where \n# := #;\n# := #." t "Program\\s-+Instance")
     ("Let" "Let" "Let # : # := #." t "Let")
-    ("Local Ltac" nil "Local Ltac # := #" t "Local\\s-+Ltac")
-    ("Ltac" "ltac" "Ltac # := #" t "Ltac")
+    ("Local Ltac2" nil "Local Ltac2 # := #." t "Local\\s-+Ltac2")
+    ("Ltac2 Type" "lt2ty" "Ltac2 Type # := #." t "Ltac2 Type")
+    ("Ltac2 Type" "lt2oty" "Ltac2 Type # ::= #." t "Ltac2 Type")
+    ("Ltac2" "ltac2" "Ltac2 # := #." t "Ltac2")
```

Comment:
Thanks @tchajed 

just a very last nitpick: we may want to replace
`("Ltac2" "ltac2" "Ltac2 # := #." t "Ltac2")`
with
`("Ltac2" "lt2" "Ltac2 # := #." t "Ltac2")`
to avoid the issue I had mentioned when typing `From Ltac2…` in abbrev-mode

Otherwise, LGTM :+1:

Reply:
Oops, sorry I missed that